src/com/goide/runconfig/testing/coverage/GoCoverageMerger.java

```diff
@@ -0,0 +1,48 @@
+package com.goide.runconfig.testing.coverage;
+
+import com.intellij.openapi.diagnostic.Logger;
+
+import java.io.*;
+
+public class GoCoverageMerger {
+  protected static final Logger LOG = Logger.getInstance("#com.goide.runconfig.testing.coverage.GoCoverageMerger");
+
+  public static void MergeCoverage(String fromPath, String toFile) {
+    try {
+      try (
+          FileWriter stream = new FileWriter(toFile, false);
+          BufferedWriter writer = new BufferedWriter(stream)
+      ) {
+        writer.write("mode: set");
+        writer.newLine();
+        mergeCoverage(writer, new File(fromPath));
+      }
+    } catch (IOException e) {
+      LOG.error(e.getMessage());
+    }
+  }
+
+  private static void mergeCoverage(BufferedWriter writer, File directory) throws IOException {
+    for (File file : directory.listFiles()) {
```

Comment:
Using VirtualFile doesn't seem to give me an up-to-date view of the file, the resulting coverage generated is empty. Ideas?